python/orca/tutorial/NCF/pytorch_predict_spark_dataframe.py

```diff
@@ -0,0 +1,88 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Step 0: Import necessary libraries
+import torch.nn as nn
+import torch.optim as optim
+
+from process_spark_dataframe import prepare_data
+from pytorch_model import NCF
+
+from bigdl.orca import OrcaContext, init_orca_context, stop_orca_context
+from bigdl.orca.learn.pytorch import Estimator
+from bigdl.orca.learn.pytorch.callbacks.tensorboard import TensorBoardCallback
+from bigdl.orca.learn.metrics import Accuracy, Precision, Recall
+
+
+# Step 1: Init Orca Context
+sc = init_orca_context(cluster_mode="local")
+spark = OrcaContext.get_spark_session()
+
+
+# Step 2: Load the processed data and configuration
+dataset_dir = "./ml-1m"
+test_data = spark.read.parquet("./test_dataframe.parquet")
+_, __, user_num, item_num, sparse_feats_input_dims, num_dense_feats, \
+    feature_cols, label_cols = prepare_data(dataset_dir)

```

Comment:
We may need to save these values in a meta file so that we don't need to compute again?